great_expectations/rule_based_profiler/estimators/kde_numeric_range_estimator.py

```diff
@@ -81,7 +81,7 @@ def _get_numeric_range_estimate(
 
         quantile_statistic_interpolation_method: str = get_quantile_statistic_interpolation_method_from_rule_state(
             quantile_statistic_interpolation_method=self.configuration.quantile_statistic_interpolation_method,
-            round_decimals=self.configuration.quantile_statistic_interpolation_method,
+            round_decimals=self.configuration.round_decimals,
```

Comment:
Does this impact any tests since we are changing this interpolation method?

Reply:
@billdirks No, because I was actually fixing a typo (we do not yet use `KDE`, so it did not manifest itself).